src/hooks/useFetchFolder.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+
+export default function useFetchFolder<T>(url: string) {
+  const [data, setData] = useState<T>();
+  const [isLoading, setIsLoading] = useState(true);
+
+  const fetchData = async (url: string) => {
+    try {
+      const response = await fetch(url);
+      const result = await response.json();
+      setData(result.folder);
+      console.log(result.folder);
+    } catch (error: unknown) {
```

Comment:
요기서는 따로 타입을 잡을 필요는 없을 것 같아요!
```suggestion
    } catch (error) {
```